### PR TITLE
Install cosign for `get-script` CI step

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind ci

#### What this PR does / why we need it:
This allows us verifying that the signature actually exist.

#### Which issue(s) this PR fixes:

None
#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: scripts/get [hunk 1]
```diff
@@ -60,7 +60,7 @@ download_binary() {
 
     mkdir -p "$(dirname "$OUTPUT")"
 
-    if command -v cosign2 >/dev/null; then
+    if command -v cosign >/dev/null; then
```
Comment (saschagrunert) [inline scripts/get:63]: Whoops, what a typo.

**Comment by saschagrunert** [comment]
CI is green:

```log
Welcome to the conmon-rs install script!
Checking if all commands are available: curl
Found cosign, verifying binary signature
tlog entry verified with uuid: b08323e5bd0dc2728ada31ea13aea8e86af525ab[9](https://github.com/containers/conmon-rs/runs/6863802856?check_suite_focus=true#step:4:10)08c53c19249dfc864e4b[10](https://github.com/containers/conmon-rs/runs/6863802856?check_suite_focus=true#step:4:11)b index: 2646964
Verified OK
Installed binary into: conmonrs
version: 0.1.0-dev
tag: none
commit: b26e0fd34b84d4c184b229b12df[11](https://github.com/containers/conmon-rs/runs/6863802856?check_suite_focus=true#step:4:12)3915fcc28ca
build: 2022-06-[13](https://github.com/containers/conmon-rs/runs/6863802856?check_suite_focus=true#step:4:14) [14](https://github.com/containers/conmon-rs/runs/6863802856?check_suite_focus=true#step:4:15):49:36 +00:00
rustc 1.61.0 (fe5b13d68 2022-05-[18](https://github.com/containers/conmon-rs/runs/6863802856?check_suite_focus=true#step:4:19))
```

File: scripts/get (hunk 1 above)
Comment (haircommander) [reply scripts/get:63]: I thought the binary was cosign2 :upside_down_face:

**Comment by saschagrunert** [comment]
Tide is gone, merging manually.

**Pull request merged**